covdefaults.py

```diff
@@ -84,6 +98,81 @@ def configure(self, config: CoverageConfig) -> None:
         if not config.get_option('report:fail_under'):
             config.set_option('report:fail_under', 100)
 
+    def fix_omit(self, config: CoverageConfig) -> None:
+        # subtract omit settings if requested
+        omit = set(config.get_option('run:omit') or [])
+        omit.add('*/setup.py')  # always ignore setup.py
+
+        # ignore common virtual environment folders, unless it's within source
+        source = config.get_option('run:source')
+        sep, suf = os.sep, ['.py', '.pyc', '.pyo']
```

Comment:
not sure why pyc and pyo are here, I don't think they should be here but I'm having a very hard time following the loop below :S

Reply:
it's because this is also what ``coverage.py`` supports and checks 🤷‍♂

Reply:
unless it actually matters, let's just use .py

Reply:
can lead to weird edge-case bugs if we use something different then what coverage.py does, but if you'd wish sure

Reply:
until there's a demonstrated problem I'd prefer the simplest thing that works